util/fusermount.c

```diff
@@ -1190,6 +1190,40 @@ static int send_fd(int sock_fd, int fd)
 	return 0;
 }
 
+static int mnt_is_ENOTCONN_for_owner(const char *mnt)
```

Comment:
Would you mind to add the explanation in the commit message as function comment? Like

/*
 * fusermount typically has the s-bit set - initial open was as root and got EACCESS
 * as 'allow_other' was not specified.
 * Try again with uid of the calling process process
 */

Reply:
Done.